spec/features/cancan_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+require 'support/helpers/login_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.